mobile/ci/mac_ci_setup.sh

```diff
@@ -55,7 +55,7 @@ do
 done
 
 # https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#xcode
-sudo xcode-select --switch /Applications/Xcode_14.1.app
+sudo xcode-select --switch "/Applications/Xcode_${XCODE_VERSION}.app"
```

Comment:
I usually have to do it like this when I'm working locally.
```suggestion
sudo xcode-select --switch "/Applications/Xcode_${XCODE_VERSION}.app/Contents/Developer"
```

Reply:
ill try that - but i think there is still a mismatch in what is setup with `rules_apple` and what is available/being provided

Reply:
Disregard, I was looking at the wrong build.